ci/test-conformance-eks.sh

```diff
@@ -218,7 +223,10 @@ function cleanup_cluster() {
        echo "=== Failed to delete EKS cluster ${CLUSTER}! ==="
        exit 1
     fi
-    rm ${KUBECONFIG_PATH}/kubeconfig
```

Comment:
nit: you can use the `-f` flag with rm, there will not be any error if the file does not exist